unit_tests/test_cluster.py

```diff
@@ -126,6 +132,145 @@ def test_search_system_suppressed_messages(self):  # pylint: disable=invalid-nam
         assert event_a["line_number"] == 6, 'Not expected event line number {}'.format(event_a["line_number"])
 
 
+def setup_test(func):
```

Comment:
This is not needed. Unit test framework does this for you.

Reply:
(actually it's pytest that doesn't it nicely in our case)

Reply:
Fixed